rdkit/ML/AnalyzeComposite.py

```diff
@@ -113,16 +112,18 @@ def ErrorStats(conn, where, enrich=1):
   if not nPts:
     sys.stderr.write('no runs found\n')
     return None
-  overall = numpy.zeros(nPts, numpy.float)
-  overallEnrich = numpy.zeros(nPts, numpy.float)
+  
+  overall = numpy.zeros(nPts, dtype='float')
```

Comment:
Here the dtype can just be (float is the default):

```
  overall = numpy.zeros(nPts)
```

or

```
 overall = numpy.zeros(nPts, dtype=float)
```

We only need to use strings representations for more complicated types